src/infobar.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import {
+  AppRegistry,
+  StyleSheet,
+  Text,
+  Image,
+  View
+} from 'react-native';
+
+export default class InfoBar extends Component {
+  constructor(props) {
+    super(props);
+  }
+  renderPicture(){
+    return(
+      <View style={styles.picture}>
+        <Image source={require('./img/bogdan.jpg')} style={{height: 30, width: 30, borderRadius: 30/2,}}/>
+      </View>
+    );
+  }
+  renderName(){
+    return(
+        <Text style={styles.nume}>Olaru Bogdan</Text>
```

Comment:
Here also, we should use a prop for the name, and pass it here from the parent. It's ok to hardcode the name in the parent, but we should use the prop here.